docs/books/admin_guide/17-log.md

```diff
@@ -501,37 +501,37 @@ Uses "[ ]" to contain the title, like the configuration files of other `systemd`
+* `SyncIntervalSec=` - Define the time interval for synchronizing the log data in memory to the disk. Please note! This only occurs for err, warning, notice, info, and debug log priorities. Other log priorities are immediately synchronized to disk. The default value is 5m.
 * `RateLimitIntervalSec=` - Define the time interval for log generation frequency. The default value is 30s.
-* `RateLimitBurst=` - The maximum number of entries generated by the log in a given time interval. The default value is 10000. If the log entries are greater than 10000 within a given time interval, it will delete the redundant logs and no generation of new log entries will occur until reaching the next time interval.
+* `RateLimitBurst=` - The maximum number of entries the log generates in a given time interval. The default value is 10000. If the log entries are greater than 10000 within a given time interval, it will delete the redundant logs, and no new log entries will be generated until the next time interval.
 * `SystemMaxUse=` - Controls the total size of all log files in the **/var/log/journal/** directory.
 * `SystemKeepFree=` - Controls how much disk space to reserve in the **/var/log/journal/** directory. Based on 1024, suffixes include K, M, G, T, P, E
 * `SystemMaxFileSize=` - Limit the size of a single file in the **/var/log/journal/** directory. If the size exceeds the specified size, log rotation will occur
 * `SystemMaxFiles=` - Specify how many files to keep in the **/var/log/journal/** directory. When it exceeds the defined number, it will delete the oldest log.
 * `RuntimeMaxUse=` - Controls the total size of log data in the **/run/log/journal/** directory.
 * `RuntimeKeepFree=` - Controls how much space to reserve in the **/run/log/journal/** directory.
 * `RuntimeMaxFileSize=` - Controls the size of a single log file in the **/run/log/journal/** directory. When the log reaches the specified size, log rotation will occur.
-* `RuntimeMaxFiles=` - How many files of logs need to keep in the **/run/log/journal/** directory.
-* `MaxRetentionSec=` - Define the retention time for log files, and if it exceeds the defined time, deleting of old log files occurs. A value of 0 indicates that the function is off. The value suffix has year, month，week，day，h，m
-* `MaxFileSec=` - Time based log rotation. Since file size based polling (`SystemMaxFileSize` and `RuntimeMaxFileSize`) already exists, time based log polling is usually not required. Set it to 0 to disable this function.
+* `RuntimeMaxFiles=` - How many files of logs need to be kept in the **/run/log/journal/** directory.
+* `MaxRetentionSec=` - Define the retention time for log files; if it exceeds the defined time, old log files are deleted. A value of 0 indicates that the function is off. The value suffix has year, month，week, day，h，m
```

Comment:
"are deleted" introduces passive voice. I'll see if I can reword it a little differently.